sk0-front/src/app/auth/auth.service.ts

```diff
@@ -102,4 +127,12 @@ export class AuthService {
     });
     await alert.present();
   }
+
+  private async showSuccessMessage(message: string) {
+    const alert = await this.alertController.create({
+      message: message,
+      buttons: ['OK'],
+    });
+    await alert.present();
+  }
```

Comment:
I am not sure why this is called `showSuccessMessage` and the other method is called `showError` that one is not a message too?

Reply:
good point, I correct it